mne/viz/topomap.py

```diff
@@ -2619,3 +2656,72 @@ def plot_arrowmap(data, info_from, info_to=None, scale=3e-10, vmin=None,
     plt_show(show)
 
     return fig
+
+
+@fill_doc
+def plot_bridged_electrodes(info, bridged_idx, ed_matrix, title=None,
+                            topomap_args=None):
+    """Topoplot electrode distance matrix with bridged electrodes connected.
+
+    Parameters
+    ----------
+    %(info_not_none)s
+    bridged_idx : list of tuple
+        The indices of channels marked as bridged with each bridged
+        pair stored as a tuple.
+    ed_matrix : ndarray of float, shape (n_channels, n_channels)
+        The electrical distance matrix for each pair of EEG electrodes.
+    title : str
+        A title to add to the plot.
+    topomap_args : dict | None
+        Arguments to pass to :func:`mne.viz.plot_topomap`.
+
+    Returns
+    -------
+    fig : instance of matplotlib.figure.Figure
+        The topoplot figure handle.
+    """
+    import matplotlib.pyplot as plt
+    if topomap_args is None:
+        topomap_args = dict()
+    if 'image_interp' not in topomap_args:
+        topomap_args['image_interp'] = 'voronoi'
+    if 'names' not in topomap_args:
```

Comment:
`.setdefault` dictionary method could be used here instead of the ifs.

Reply:
Either way dict is mutable and we don't want to change a user's input so if they do pass a dict you should .copy() it before doing anything to it